main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       bird
+//  action 1:   the bird flies
+bird.fly();
+//  action 2:   the bird sings
+bird.sing();
+//  action 3:   the bird pecks wood
+bird.peckWood();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       baby
+//  action 1:   the baby cries
+baby.cry();
+//  action 2:   the baby smiles
+baby.smile();
+//  action 3:   the baby sleeps
+baby.sleep();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog runs
+dog.run();
+//  action 3:   the dog drinks water
+dog.drinkWater();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       sun
+//  action 1:   the sun rises up
+sun.riseUp();
+//  action 2:   the sun goes down
+sun.goDown();
+//  action 3:   the sun shines light
+sun.shineLight();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       oven
+//  action 1:   the oven reaches temperature
+oven.reachTemperature();
+//  action 2:   the oven roasts chicken
+oven.roastChicken();
+//  action 3:   the oven burns food
+oven.burnFood();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       car
+//  action 1:   the car brakes
+car.brake();
+//  action 2:   the car turns left
+car.turnLeft();
+//  action 3:   the car speeds up
+car.speedUp();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.